tests/test_create_subgroup.py

```diff
@@ -1,45 +1,10 @@
 from unittest.mock import Mock
 
 import pytest
-from sqlalchemy import create_engine
-from sqlalchemy.orm import sessionmaker, Session
+from sqlalchemy.orm import Session
 
-from shout_subgroup.models import Base, UserModel, GroupChatModel
 from shout_subgroup.create_subgroup import create_subgroup
-
```

Comment:
Logic no longer needed b/c of the new helper functions and db being moved into common file